src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getAllUsers() {
+  const response = await fetch(`${BASE_URL}/users`);
+
+  return response.json();
```

Comment:
const response can be dismissed:
![image](https://github.com/pedro-ruas/mate-react_dynamic-list-of-posts_test-task/assets/54222238/5459b1ce-17fd-40c5-94ab-5437dae367d0)